fem/bilininteg_mass_pa.cpp

```diff
@@ -1208,8 +1218,8 @@ static void PAMassApply(const int dim,
          case 0x9A: return SmemPAMassApply3D<9,10>(NE,B,Bt,D,X,Y);
          default:   return PAMassApply3D(NE,B,Bt,D,X,Y,D1D,Q1D);
       }
+      mfem::out << "Unknown 3D kernel 0x" << std::hex << id << std::endl;
```

Comment:
Unreachable?

Reply:
right, I'll revert in the other PR.

Reply:
`done`

Reply:
Why in another PR? This is introduced here, so it should be removed here.

Reply:
I think this code has been moved to #1735